eva_sub_cli_processing/sub_cli_to_eload_converter/json_to_xlsx_converter.py

```diff
@@ -0,0 +1,231 @@
+        "description": "Description",
+        "centre": "Center",
+        "taxId": "Tax ID",
+        "publications": "Publication(s)",
+        "parentProject": "Parent Project",
+        "childProjects": "Child Project(s)",
+        "peerProjects": "Peer Project(s)",
+        "links": "Link(s)",
+        "holdDate": "Hold Date",
+        "collaborators": "Collaborator(s)",
+        "strain": "Strain",
+        "breed": "Breed",
+        "broker": "Broker"
+    },
+
+    "analysis": {
+        "analysisTitle": "Analysis Title",
+        "analysisAlias": "Analysis Alias",
+        "description": "Description",
+        "experimentType": "Experiment Type",
+        "referenceGenome": "Reference",
+        "referenceFasta": "Reference Fasta Path",
+        "platform": "Platform",
+        "software": "Software",
+        "pipelineDescriptions": "Pipeline Description",
+        "imputation": "Imputation",
+        "phasing": "Phasing",
+        "centre": "Centre",
+        "date": "Date",
+        "links": "Link(s)",
+        "runAccessions": "Run Accession(s)"
+    },
+
+    "sample": {
+        "analysisAlias": "Analysis Alias",
+        "sampleInVCF": "Sample Name in VCF",
+        "bioSampleAccession": "Sample Accession",
+        "bioSampleName": "BioSample Name",
+        "title": "Title",
+        "description": "Description",
+        "uniqueName": "Unique Name",
+        "prefix": "Prefix",
+        "subject": "Subject",
+        "derivedFrom": "Derived From",
+        "taxId": "Tax Id",
+        "scientificName": "Scientific Name",
+        "commonName": "Common Name",
+        "matingType": "mating_type",
+        "sex": "sex",
+        "population": "population",
+        "cellType": "cell_type",
+        "devStage": "dev_stage",
+        "germline": "germline",
+        "tissueLib": "tissue_lib",
+        "tissueType": "tissue_type",
+        "BioMaterial": "bio_material",
+        "cultureCollection": "culture_collection",
+        "specimenVoucher": "specimen_voucher",
+        "collectedBy": "collected_by",
+        "collectionDate": "collection_date",
+        "geographicLocationCountrySea": "geographic location (country and/or sea)",
+        "geographicLocationRegion": "geographic location (region and locality)",
+        "host": "host",
+        "identifiedBy": "identified_by",
+        "isolationSource": "isolation_source",
+        "latLon": "lat_lon",
+        "LabHost": "lab_host",
+        "environmentalSample": "environmental_sample",
+        "cultivar": "cultivar",
+        "ecotype": "ecotype",
+        "isolate": "isolate",
+        "strain": "strain",
+        "subSpecies": "sub_species",
+        "variety": "variety",
+        "subStrain": "sub_strain",
+        "cellType": "cell_line",
+        "serotype": "serotype",
+        "serovar": "serovar"
+    },
+
+    "files": {
+        "analysisAlias": "Analysis Alias",
+        "fileName": "File Name"
+    }
+}
+
+
+class JsonToXlsxConverter:
```

Comment:
Have you tried reusing the [EvaXlsxWriter](https://github.com/EBIvariation/eva-submission/blob/18dcf121be1a82511ea3911a9a121893cb1213e1/eva_submission/xlsx/xlsx_parser_eva.py#L90C1-L91C1) ?